### PR TITLE
Modifications to event registry tests

### PR DESCRIPTION
**Pull request by tpantelis**
See commits for details.

### CODE REVIEW

File: pkg/event/registry_test.go [hunk 1]
```diff
@@ -17,95 +17,91 @@ const npOVNKubernetes = "OVNKubernetes"
...
-				Expect(handler1.Events).Should(Receive(Equal(ev)))
-				Expect(handler2.Events).Should(Receive(Equal(ev)))
-
-				// Handlers should be handled invoked in order of registration
-				ev.Handler = handler1.Name
-				Expect(allTestEvents).Should(Receive(Equal(ev)))
-				ev.Handler = handler2.Name
-				Expect(allTestEvents).Should(Receive(Equal(ev)))
+
+				for _, h := range matchingHandlers {
```
Comment (mangelajo) [inline pkg/event/registry_test.go:62]: In this unit testing it's not implicit that we expect that the handlers should be called in order. I'd add a comment at least.

Imagining that someone rewrites the registry (somehow.. with a map...) .. and then the order is broken, so they change this test here.

If we say explicitly somewhere here that the events of the handlers should be received in order ... it'd safer I think

**Review by mangelajo — CHANGES_REQUESTED** [review]
Only a small comment, not sure how to do it best, just comments in the code, or extension of the "It" item.

File: pkg/event/registry_test.go (hunk 1 above)
Comment (tpantelis) [reply pkg/event/registry_test.go:62]: yeah - that is already described in the registry `AddHandlers` API - "_Handlers will be called in registration order"_. But I add it to the _It_ text as well.

**Review by mangelajo — APPROVED** [review]
Thanks !! :-)

**Review by sridhargaddam — APPROVED** [review]

**Pull request merged**